client/blocks/login/index.jsx

```diff
@@ -140,11 +143,29 @@ class Login extends Component {
 					clientTitle: oauth2ClientData.title
 				}
 			} );
+			if ( oauth2ClientData === 'woo' ) { //TODO
```

Comment:
We need to check the client id, which will be easier once this patch is merged: https://github.com/Automattic/wp-calypso/compare/add/oauth-login-ssr...add/oauth-login-ssr-2?expand=1